include/mlir/Dialect/SPIRV/SPIRVOps.td

```diff
@@ -66,8 +66,11 @@ def SPV_AccessChainOp : SPV_Op<"AccessChain", [NoSideEffect]> {
     no remaining (unused) indexes.
 
      Each index in Indexes
+
```

Comment:
Nit : Why the extra line here?

Reply:
It's a autofix by define_inst.sh script, which fixes my previous mistake, I've removed those lines by hands.